### PR TITLE
Fix ComboBox maxWidth shifting around when using virtualization

### LINKED ISSUE #955: Combobox : Improvement in dropdown menu width when enabledVirtulization is true
#### Feature
When combobox has enbled virtulization is true then dropdown menu width changes depending on the width of menu. This can be improved by setting dropdownmenu with to combobox width if virtualization is enabled.

#### Examples
You can see in below example dropdown width is changing.
combobox

### PR DESCRIPTION
**Pull request by mayank99**
Fixes #955

### CODE REVIEW

File: packages/itwinui-react/src/core/ComboBox/ComboBoxMenu.tsx [hunk 1]
```diff
@@ -61,7 +61,10 @@ const VirtualizedComboBoxMenu = React.forwardRef(
 
     const surfaceStyles = {
       minWidth,
-      maxWidth: `min(${minWidth * 2}px, 90vw)`,
+
+      // set as constant because we don't want it shifting when items are unmounted
+      maxWidth: minWidth,
```
Comment (nigtas) [inline packages/itwinui-react/src/core/ComboBox/ComboBoxMenu.tsx:66]: Have you checked how it looks like if virtualization is enabled and item which is not rendered initially is much longer than the width was set before...?

Comment (veekeys) [inline packages/itwinui-react/src/core/ComboBox/ComboBoxMenu.tsx:66]: Have you checked how it looks when virtualization is enabled and the item which is not rendered initially is much wider than the calculated width?

Comment (veekeys) [reply packages/itwinui-react/src/core/ComboBox/ComboBoxMenu.tsx:66]: oh ok.. so it just never recalculates and hardcodes the width as it is calculated from the initial visible list.

**Review by FlyersPh9 — APPROVED** [review]

**Review by adamhock — APPROVED** [review]

**Pull request merged**